src/components/AdminPage/AdminHome/AdminHome.js

```diff
@@ -1,5 +1,39 @@
+import { useState } from "react";
+import { auth } from "../../../firebase";
+import { useNavigate } from "react-router-dom";
+import { signOut } from "firebase/auth";
+
 const AdminHome = () => {
-  return <div>AdminHome</div>;
+  const [isSignedIn, setIsSignedIn] = useState(false);
+
+  const navigate = useNavigate();
+  const navigation = () => {
+    navigate("/admin");
+  };
+
+  const signOutUser = () => {
+    signOut(auth)
+      .then((res) => {
+        setIsSignedIn(true);
+        navigation();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
```

Comment:
Please use async/await. In catch we need to add something to inform user about error.